components/member-profile/index.js

```diff
@@ -13,6 +13,42 @@ const Profile = (props) => {
   const memberName = `${first_name} ${last_name}`;
   const socialMedia = ['twitter_id', 'github_id', 'linkedin_id', 'instagram_id'];
 
+  const [pullRequests, setPullRequests] = useState([]);
+  useEffect(() => {
+    getPRsbyUser(id).then((res) => setPullRequests(res));
+  }, []);
+
+  console.log(pullRequests);
+  const showPRdetails = pullRequests.map((obj) => {
+    let reviewState;
+    if (obj.readyForReview) {
+      reviewState = 'Ready for review';
+    } else if (obj.state === 'closed') {
+      reviewState = '';
+    } else {
+      reviewState = 'Changes requested';
+    }
```

Comment:
NIT: If it is anyway the default `reviewState`, you can directly declare the `reviewState` variable like this,
`let reviewState = 'Changes requested';`

Reply:
how can the default value be `Changes requested` ?

Reply:
If you parse through the `if-else` statement, the default value will be `Changes requested`